dashboards/Advanced_Data_Exploration.json

```diff
@@ -670,7 +670,7 @@
             {
                 "allValue": null,
                 "current": {},
-                "datasource": "Prometheus",
+                "datasource": "Metrics",
```

Comment:
Can we document cases when the `datasource` takes the value of `null` vs `Metrics`.